domenetjenester/uttak/src/main/java/no/nav/foreldrepenger/domene/uttak/uttaksgrunnlag/fp/FastsettUttaksgrunnlagTjeneste.java

```diff
@@ -108,6 +107,49 @@ private OppgittFordelingEntitet justerFordeling(UttakInput input, LocalDate endr
         return new OppgittFordelingEntitet(kopier(justertePerioder), fordeling.getErAnnenForelderInformert(), fordeling.ønskerJustertVedFødsel());
     }
 
+    private static boolean skalJustereFordelingEtterFamiliehendelse(ForeldrepengerGrunnlag fpGrunnlag, List<OppgittPeriodeEntitet> perioder, Long behandlingId) {
+        if (behandlingId.equals(3066799L)) { // TODO: Fjern denne etter behandling OK.
```

Comment:
Kunne code reviewers verifisert at det er behandlingsid 3066799 som skal slippes igjennom for saksnummer 152347615?